src/applications/hca/config/form.js

```diff
@@ -397,6 +397,7 @@ const formConfig = {
             veteranAddress: _.merge(addressUI('Mailing address', true), {
               'ui:description': <AddressDescription addressType="mailing" />,
               street: {
+                'ui:title': 'Street address',
```

Comment:
When adding a `ui:title`, it adds a wrapping `fieldset` with a `legend`, which may not be appropriate in this case. If you do find that problem then you may need to add a `forceDIvWrapper: true` to the `ui:options` to prevent it. For optimal accessibility, we want one wrapping `fieldset` around the address block (with a non-empty legend), and a `label` associated with each input - no non-associated labels.

Reply:
Does not look like its an issue on this component in particular but thanks for letting me know will keep a look out in the future. Probably need to go through the 1010CG and look for this issue. I tried to run WAVE on this page as you recommended but unfortunately since the form is gate kept by a intro form WAVE cant run the actual form part of this URL.  

![image](https://user-images.githubusercontent.com/23741323/106041368-b5287880-60a9-11eb-8b13-7dbd8ad6ab17.png)